lib/charty/table_adapters/active_record_adapter.rb

```diff
@@ -0,0 +1,48 @@
+module Charty
+  module TableAdapters
+    class ActiveRecordAdapter
+      include Enumerable
+
+      def self.make(data)
+        self.new(data)
+      end
+
+      def self.supported?(data)
+        defined?(ActiveRecord::Relation) && data.is_a?(ActiveRecord::Relation)
+      end
+
+      def initialize(data)
+        @data = check_type(ActiveRecord::Relation, data, :data)
+        @columns = @data.column_names.freeze
+        @arrays = nil
+      end
+
+      attr_reader :columns
+
+      def column(i)
```

Comment:
How about `column_data` or `column_values`?
I like `fetch_column_XXX` or something but it's long...

Is `#[](column_index)` enough?